frontend/package.json

```
@@ -23,8 +25,9 @@
   "scripts": {
     "start": "DISABLE_ESLINT_PLUGIN=true REACT_APP_BACKEND_DOMAIN=http://localhost:8080 react-scripts start",
     "build": "DISABLE_ESLINT_PLUGIN=true react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --testPathIgnorePatterns=.*e2e.test.js",
```

This can be trimmed down since we don't have any `e2e.test.js` files.

```suggestion
    "test": "react-scripts test",
```